appendices/benchmark_results.md

```diff
@@ -20,7 +20,7 @@ Vector already did benchmark tests for the performance of different APM tools in
 We stick to the results obtained by Vector and extend the results table by adding Edge Delta agent results. Therefore, the table of the results took the shape below.
 ---
 description: >-
-make sure this is kept in sync with the table on https://edgedelta.com/. If new benchmark results are calculated ping Ozan to update https://edgedelta.com/
+  make sure this is kept in sync with the table on https://edgedelta.com/. If new benchmark results are calculated ping Ozan to update https://edgedelta.com/
```

Comment:
we don't want this comment to be visible to users. use markdown comments https://stackoverflow.com/questions/4823468/comments-in-markdown

Also some parts of the text are bold where it shouldn't be

![Screenshot - 2021-05-25T093344 577](https://user-images.githubusercontent.com/1688634/119535035-71363f00-bd3c-11eb-9e64-00bb63d62fda.png)

Reply:
I applied [comment]: <> (This is a comment, it will not be included) an it worked